<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentationPropertiesWidget_CustomSlide</class>
 <widget class="QWidget" name="PresentationPropertiesWidget_CustomSlide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>249</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="pixmap">
      <pixmap resource="../../../res/resources.qrc">:/icons/16/Align Text Left_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="pixmap">
      <pixmap resource="../../../res/resources.qrc">:/icons/16/Full Image_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pozad√≠:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="PresentationBackgroundSelectionWidget" name="wgtBackground" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="teText"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Styl:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="pixmap">
      <pixmap resource="../../../res/resources.qrc">:/icons/16/Paint Palette_16px.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="StyleSelectionWidget" name="wgtStyle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyleSelectionWidget</class>
   <extends>QWidget</extends>
   <header>wgt/styleselectionwidget.h</header>
   <container>1</container>
   <slots>
    <signal>sigPresentationStyleChangedByUser()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>PresentationBackgroundSelectionWidget</class>
   <extends>QWidget</extends>
   <header>wgt/presentationbackgroundselectionwidget.h</header>
   <container>1</container>
   <slots>
    <signal>sigPresentationBackgroundChangedByUser(PresentationBackground)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>wgtStyle</tabstop>
  <tabstop>wgtBackground</tabstop>
  <tabstop>teText</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
